### PR TITLE
Design/lazy plugin renderer

### PR DESCRIPTION
**Pull request by lcarva**

### CODE REVIEW

File: docs/build_parameters.rst [hunk 1]
```diff
@@ -254,10 +252,37 @@ to the root of the object::
...
+                      "name": {
+                        "description": "signing intent name",
+                        "type": "string"
+                      },
+
+                      "keys": {
+                          "description": "signing intent keys",
+                          "type": "array",
+                          "items": {
+                              "type": "string"
```
Comment (csomh) [inline docs/build_parameters.rst]: Could this be more restrictive? Like hexadecimal?

File: docs/build_parameters.rst [hunk 2]
```diff
@@ -254,10 +252,37 @@ to the root of the object::
...
                     }
                 ]
+            },
+            "signing_intents": {
+                "description": "List of signing intents",
+                "type": "array",
+                "items": {
+                    "properties": {
+                      "name": {
+                        "description": "signing intent name",
```
Comment (csomh) [inline docs/build_parameters.rst]: Could this be made uniform, and start descriptions with a capital letter? (And not ending them with a period.)

File: docs/build_parameters.rst (hunk 1 above)
Comment (twaugh) [reply docs/build_parameters.rst]: `"pattern": "^[a-fA-F0-9]+$"` ?

**Comment by lcarva** [comment]
Removed the schema completely out of design doc. It now references the one in atomic-reactor itself.
Added various changes to bring design doc up to speed.

**Comment by lcarva** [comment]
Reviewers, ping.

**Pull request merged**